src/jrd/dfw.epp

```diff
@@ -3549,6 +3549,19 @@ static bool create_relation(thread_db* tdbb, SSHORT phase, DeferredWork* work, j
 	switch (phase)
 	{
 	case 0:
+		// We need to cleanup RDB$PAGES and pages if their were added at phase 3.
+		request.reset(tdbb, irq_c_relation2, IRQ_REQUESTS);
```

Comment:
I think it's too risk to reuse a IRQ constant just because you compared the existing query and accessed fields and their are the same. Please create a new constant for the new use.